Makefile

```diff
@@ -153,16 +153,12 @@ unit-tests:
 
 .PHONY: e2e-tests
 
-e2e-tests: test-e2e-setup
+e2e-tests:
 	@echo "Running E2E Tests.."
 	@./cicd-scripts/run-e2e-tests.sh
 
 test-e2e-setup:
 	@echo "Seting up E2E Tests environment..."
-ifdef COMPONENT_IMAGE_PIPELINE
-	# override the image for the e2e test
-	@./cicd-scripts/setup-e2e-tests.sh -a install -p $(COMPONENT_IMAGE_PIPELINE)
-endif
 ifdef COMPONENT_IMAGE_NAMES
 	# override the image for the e2e test
 	@./cicd-scripts/setup-e2e-tests.sh -a install -i $(COMPONENT_IMAGE_NAMES)
```

Comment:
These changes are trying to make the e2e testing in prow build work, what about the local test?
I think we also need to setup the e2e testing env locally, maybe in future PRs.

Reply:
`COMPONENT_IMAGE_PIPELINE` was introduced to get all images from this pipeline. it is not available in `ocm-e2e-clusterpool`. `COMPONENT_IMAGE_NAMES` is still there for local tests. But you are correct, I need to have a try with local testing.